tests/PhpSpreadsheetTests/Worksheet/DrawingTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace PhpOffice\PhpSpreadsheetTests\Worksheet;
+
+use PhpOffice\PhpSpreadsheet\Reader\Xls;
+use PHPUnit\Framework\TestCase;
+
+class DrawingTest extends TestCase
+{
+    public function testCloningWorksheetWithImages()
+    {
+        $filename = './data/Reader/Xls/with_images.xls';
```

Comment:
Please refrain from adding binary files to the repo and instead create them from scratch, in memory with PhpSpreadsheet.